src/client/ContentManagement.js

```diff
@@ -749,10 +749,11 @@ exports.CreateNonOwnerCap = async function({objectId, libraryId, publicKey, writ
  * @param {object=} options -
  * meta: New metadata for the object - will be merged into existing metadata if specified
  * type: New type for the object - Object ID, version hash or name of type
+ * @param {string} httpNodeUrl - Node URL to use in HTTP call
```

Comment:
optional? if so, should be ` * @param {string=} `  (or could make a field of `options` param instead)